apps/cowswap-frontend/src/modules/swap/helpers/getAmountsForSignature.test.ts

```diff
@@ -67,17 +81,23 @@ describe('getAmountsForSignature()', () => {
       // Just subtracted fee from inputAmount, 3012 - 8 = 3004
       expect(result.inputAmount.toFixed()).toEqual('3004.000000')
 
-      // Fee in WETH = 0.005312084993359893
-      // outputAmount = (2 - 0.005312) - 5% = 1.89495
-      expect(result.outputAmount.toFixed()).toEqual('1.894953519256308100')
+      if (partnerFeeBps === 0) {
+        // outputAmount = 2 - 5% = 1.900000000000000000
+        expect(result.outputAmount.toFixed()).toEqual('1.900000000000000000')
+      } else {
+        // outputAmount = 2 - 5% - 20% = 1.520000000000000000
+        expect(result.outputAmount.toFixed()).toEqual('1.520000000000000000')
+      }
```

Comment:
Here too. These are 2 different tests to me.

Reply:
Fixed